Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.py

```diff
@@ -3037,28 +3071,81 @@ def update_case_command(client: Client, args: dict) -> CommandResults:
         "resolve_reason": CaseManagement.STATUS_RESOLVED_REASON.get(resolve_reason) if resolve_reason else None,
         "caseResolvedComment": resolved_comment if resolved_comment else None,
         "resolve_all_alerts": resolve_all_alerts if resolve_all_alerts else None,
-        "CustomFields": custom_fields if custom_fields else None,
+        "CustomFields": valid_fields_to_update if valid_fields_to_update else None,
     }
     remove_nulls_from_dictionary(case_update_payload)
 
     if not case_update_payload and args.get("assignee", "").lower() != "unassigned":
-        raise ValueError("No valid update parameters provided for case update.")
+        raise ValueError(f"No valid update parameters provided.\n{error_messages}")
 
-    demisto.info(f"Executing case update for cases {case_ids} with request data: {case_update_payload}")
     responses = [client.update_case(case_update_payload, case_id) for case_id in case_ids]
-    replies = []
-    for resp in responses:
-        replies.append(process_case_response(resp))
+    replies = [process_case_response(resp) for resp in responses]
+    readable_output = tableToMarkdown("Cases", replies, headerTransform=string_to_table_header)
+    entry_type = entryTypes["note"]
+    if error_messages:  # If some fields failed, we use the error entry type
+        entry_type = entryTypes["error"]
+        unsuccessful_section = f"Case updated successfully. The following fields could not be updated:\n" f"{error_messages}"
```

Comment:
I think better to separate to a regular entry and separately call `return_error(error_messages)`